src/utils/utils.ts

```diff
@@ -15,3 +19,24 @@ export const axiosBuilder = (axiosFunc: Promise<unknown>, text: string, router,
       showSnackbar(error.response.data.message, 'error');
     });
 };
+
+export const toggleCheckBox = (
+  employeeTask: IEmployeeTask,
+  completed: boolean,
+  setCompleted: Dispatch<SetStateAction<boolean>>,
+  showSnackbar: (arg0: string, arg1: string) => void,
+) => {
+  axios
+    .put(`/api/employeeTasks/${employeeTask.id}`, {
+      completed: !completed,
+      dueDate: employeeTask.dueDate,
+      responsibleId: employeeTask.responsibleId,
+    })
+    .then(() => {
+      showSnackbar(`Oppgave ${completed ? 'ikke' : ''} fullført`, 'success');
```

Comment:
Litt pirk, men synes kanskje det er litt mer beskrivende å si at:

```suggestion
      showSnackbar(`Oppgave markert som  ${completed ? 'ikke' : ''} fullført`, 'success');
```